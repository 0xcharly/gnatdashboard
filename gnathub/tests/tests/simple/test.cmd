# launch GNAThub on a trivial project
gnathub -P default

# verify that the object directory contains the expected files
# at the expected locations
find obj/ | grep -v ".svn" | sort
