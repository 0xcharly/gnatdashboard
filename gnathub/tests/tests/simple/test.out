obj/
obj/hello.adb.metrix
obj/metrix.xml
obj/gnathub
obj/gnathub/logs
obj/gnathub/logs/gnatmetric.log
obj/gnathub/logs/sonar-runner.log
obj/gnathub/qualimetrics.db
obj/gnathub/sonar
obj/gnathub/sonar/sonar-project.properties
